hadoop-hdfs-project/hadoop-hdfs-rbf/pom.xml

```diff
@@ -177,11 +177,6 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
-      <dependency>
```

Comment:
sorry, i meant "align the elements with the ones above"

now, is this going to be needed or is it coming down from the other dependencies anyway (which I would expect). 

we will find out from the result of the build; if all compiles then mockito core is pulled in by mockito.

we've been burned a few times by people (myself included) not realising that when a -test jar is pulled in by another module, h of the test jars dependencies are transitively picked up. this has made rollout of assertJ harder than we'd like. Let's see what happens here

Reply:
This mockito-core dependency was already present in this file, and got added again as a part of the previous commit with wrong alignment. Have removed the duplicate entry.